docs/source/development/redirect-after-logout.md

```diff
@@ -0,0 +1,37 @@
+---
+myst:
+  html_meta:
+    "description": "Redirect to the login page after logout in Volto"
+    "property=og:description": "How to redirect to the login page after logout in Volto"
+    "property=og:title": "Redirect to the login page after logout in Volto"
+    "keywords": "Volto, Plone, frontend, React, logout, redirect"
+---
+
+# Redirect to the login page after logout
+
+
+To redirect users to the login page after they log out in Volto, you need to customize the `PersonalTools` component.
+Follow these steps.
+
+1.  **Shadow the component**: Create a new file at `customizations/components/manage/Toolbar/PersonalTools.jsx`, then copy the contents of the original file from {file}`volto/packages/volto/src/components/manage/Toolbar/PersonalTools.jsx` into the new file.
```

Comment:
I think the best place to do so is in the View, rather than customizing the component as a whole. For example we do redirect to login when unauthorized https://github.com/eea/volto-eea-website-theme/blob/ff514ef4cd7833edcf3956f300092554a0d8ac2e/src/customizations/volto/components/theme/Unauthorized/Unauthorized.jsx#L28